libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
Ah... got you now. I hadn't considered the state of `errno` after `__MCSFileSetContents` returns... indenting was Xcode. It looks indented correctly there...